tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraForwardSetMethodContract.scala

```diff
@@ -0,0 +1,1028 @@
+         |  "methodResponses": [
+         |    ["Forward/set", {
+         |      "accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+         |      "newState": "${INSTANCE.value}",
+         |      "notDestroyed": {
+         |        "singleton": {
+         |          "type": "invalidArguments",
+         |          "description": "'destroy' is not supported on singleton objects"
+         |        }
+         |      }
+         |    }, "c1"]
+         |  ]
+         |}""".stripMargin)
+  }
+
+  @Test
+  def messageShouldBeForwardedToDestinationForwards(server: GuiceJamesServer): Unit = {
+    val request: String =
+      s"""{
+         |    "using": [ "urn:ietf:params:jmap:core",
+         |               "com:linagora:params:jmap:forward" ],
+         |    "methodCalls": [
+         |      ["Forward/set", {
+         |        "accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+         |        "update": {
+         |            "singleton": {
+         |                "localCopy": true,
+         |                "forwards": [ "${ANDRE.asMailAddress().asString()}"]
+         |            }
+         |        }
+         |      }, "c1"]
+         |    ]
+         |  }""".stripMargin
+
+    `given`
+      .body(request)
+    .when
+      .post
+    .`then`
+      .statusCode(SC_OK)
+      .contentType(JSON)
+
+    val mail: FakeMail = FakeMail.builder()
+      .name("mail1")
+      .mimeMessage(MimeMessageBuilder.mimeMessageBuilder()
+        .setSender(BOB.asString())
+        .addToRecipient(BOB.asString())
+        .setSubject("Subject 01")
+        .setText("Content mail 123"))
+      .sender(BOB.asString())
+      .recipient(BOB.asString())
+      .build()
+
+    new SMTPMessageSender(DOMAIN.asString())
+      .connect("127.0.0.1", server.getProbe(classOf[SmtpGuiceProbe]).getSmtpPort)
+      .authenticate(BOB.asString(), BOB_PASSWORD)
+      .sendMessage(mail)
+
+    baseRequestSpecBuilder(server)
+      .setAuth(authScheme(UserCredential(BOB, BOB_PASSWORD)))
+      .addHeader(ACCEPT.toString, ACCEPT_RFC8621_VERSION_HEADER)
+      .build
+
+    CALMLY_AWAIT.atMost(30, TimeUnit.SECONDS).untilAsserted { () =>
+      assertThat(listAllMessageResult(server, ANDRE)).hasSize(1)
+    }
+  }
+
+  @Test
+  def messageShouldBeForwardedToOwnerWhenLocalCopyIsTrue(server: GuiceJamesServer): Unit = {
+    assertThat(listAllMessageResult(server, BOB)).hasSize(0)
+    val request: String =
+      s"""{
+         |    "using": [ "urn:ietf:params:jmap:core",
+         |               "com:linagora:params:jmap:forward" ],
+         |    "methodCalls": [
+         |      ["Forward/set", {
+         |        "accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+         |        "update": {
+         |            "singleton": {
+         |                "localCopy": true,
+         |                "forwards": []
```

Comment:
Your forwards are empty here, of course the owner would end up with the message... I would expect more a test with at least one forward and localCopy true and making sure it ends up in the owner mailbox instead here?